fs/localfs/local_fs_windows.go

```diff
@@ -9,3 +9,7 @@ import (
 func platformSpecificOwnerInfo(fi os.FileInfo) fs.OwnerInfo {
 	return fs.OwnerInfo{}
 }
+
+func platformSpecificDeviceInfo(fi os.FileInfo) fs.DeviceInfo {
```

Comment:
Windows allows composing arbitrary FS trees through a couple of mechanisms. I don't have much experience here, but I'm assuming that is a "rare" use case, so it is OK to not support it for now.

Refs:
* https://docs.microsoft.com/en-us/windows/win32/fileio/volume-mount-points
* https://docs.microsoft.com/en-us/windows/win32/fileio/reparse-points-and-file-operations

Reply:
Well, this does give me an idea of a missing feature: the one-file-system-policy should be refused in unsupported platforms with an error, or at the very least it should prompt a warning; possibly when setting the policy or when applying it.

The support could be introduced when Kopia learns more Windows tricks, such as using shadow volumes.